src/providers/firestore.ts

```diff
@@ -70,12 +71,12 @@ export class NamespaceBuilder {
 
 export interface DeltaDocumentSnapshot {
   exists: Boolean;
-  ref: any;
+  ref: firestore.DocumentReference;
   id: string;
-  createTime: string;
-  updateTime: string;
-  readTime: string;
-  previous: any;
+  createTime: string | null;
```

Comment:
Can you change the cases of "field: string | null" to be "field?: string"

Reply:
@laurenzlong Changed. Thanks! :)